CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 v1.6.1 [unreleased]
 -------------------
 
+### Features
+-   [#246](https://github.com/latera/homs/pull/246) Compile widget with Font Awesome.
```

Comment:
Let's be honest, widget is not _compiled_ with FontAwesome now, widget will download it when page loaded.